lib/vrl/parser/build.rs

```diff
@@ -4,6 +4,7 @@ fn main() {
     println!("cargo:rerun-if-changed=src/parser.lalrpop");
     lalrpop::Configuration::new()
         .always_use_colors()
+        .emit_whitespace(false)
```

Comment:
Neat.

Reply:
Minimally changes the parser build time, but felt like a freebie so why not.

Reply:
100%